infrastructures/openshift/src/main/java/org/eclipse/che/workspace/infrastructure/openshift/provision/Openshift4TrustedCAProvisioner.java

```diff
@@ -67,9 +79,22 @@ public void provision(KubernetesEnvironment k8sEnv, OpenShiftProject project)
     if (!trustedStoreInitialized) {
       return;
     }
+    ConfigMap configMap =
+        clientFactory
+            .createOC()
+            .configMaps()
+            .inNamespace(cheInstallationLocation.getInstallationLocationNamespace())
```

Comment:
Can we do this installation namespace evaluation just once in constructor?

Reply:
Well, then we should enable throwing an `InfrastructureException` from the constructor

Reply:
Its not a problem, i mean throwing exc from constructor, we already did that in some other classes.

Reply:
OK, let me to the change then.